compiler/rustc_codegen_cranelift/src/driver/aot.rs

```diff
@@ -487,6 +503,7 @@ fn reuse_workproduct_for_cgu(
             bytecode: None,
             assembly: None,
             llvm_ir: None,
+            links_from_incr_cache: links_from_incr_cache.clone(),
```

Comment:
I think both `CompiledModule` should use a separate `links_from_incr_cache`.

Reply:
Oh :facepalm: I see now